dotCMS/src/main/java/com/dotmarketing/portlets/widget/business/WidgetAPIImpl.java

```diff
@@ -19,8 +21,9 @@
 
 public class WidgetAPIImpl implements WidgetAPI {
 	
-	public PermissionAPI perAPI = APILocator.getPermissionAPI();
+	public final PermissionAPI perAPI = APILocator.getPermissionAPI(); // todo: could it be private?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)